src/Files.App/ViewModels/UserControls/AddressToolbarViewModel.cs

```diff
@@ -688,6 +688,10 @@ public async Task SetPathBoxDropDownFlyoutAsync(MenuFlyout flyout, PathBoxItem p
 
 		private static string NormalizePathInput(string currentInput, bool isFtp)
 		{
+			// Check if input is a command and not a path
+			if (currentInput.Length > 1 && currentInput.ElementAt(1) != ':')
```

Comment:
Is `:` always the second character in a valid path?

Reply:
Couldn't the paths be relative? `./Child` `../Sibling` etc? It won't have a drive letter then

Reply:
I'm not concerned since we don't have plans on supporting that, but worst case we can come up with a new fix if things change.